CSSIntro.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>CSS Intro</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css?family=Vibes&display=swap');
```

Comment:
This is not wrong, but just letting you know you can import this on the css file as well, but pasting it on top.